### PR TITLE
refactor: promo scss + examples scss

### PR DESCRIPTION
**Pull request by ossset**

### CODE REVIEW

File: sass/partials/_about-us.scss [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+        font-size: 60px;
+    }
+    
+    &__quote-text {
+    
+        padding: 17px 0 35px 0;
+        text-align: center;
+    
+        font-size: 13px;
+        font-style: italic;
```
Comment (uladzislau-kirychenka) [inline sass/partials/_about-us.scss]: It's better to use <I> tag

File: sass/partials/_care.scss [hunk 2]
```diff
@@ -0,0 +1,218 @@
+.care { 
+    & {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_care.scss]: redundant style block

File: sass/partials/_examples.scss [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+    &__card {
+        position: relative;
+    
+        width: 280px;
+        height: 120px;
+        margin: 0 auto 10px auto;
+        padding: 13px 0 0 14px;
+    }
+    
+    &__name::after {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_examples.scss]: move it into name element style block

File: sass/partials/_examples.scss [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+        content: "";
+    
+        background-color: rgba(0, 0, 0, 0.1);
+    }
+    
+    &__card:hover &__name::after {
+        display: block;
+    }
+    
+    &__card--classic {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_examples.scss]: Move all modifiers into card element style block

File: sass/partials/_care.scss [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+            right: -25px;
+            width: 780px;
+            height: 805px;
+        }
+        &__algorithm{
+            background: #c93030;
+            position: relative;
+            height: 700px;
+        }
+        &__algorithm-wrapper::after{
```
Comment (uladzislau-kirychenka) [inline sass/partials/_care.scss]: move it to algorithm wrapper

File: sass/partials/_about-us.scss [hunk 6]
```diff
@@ -0,0 +1,265 @@
...
+        position: relative;
+    
+        text-decoration: underline;
+    
+        color: #f87b91;
+    
+        font-style: italic;
+    }
+    
+    &__feedback-link::after {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_about-us.scss]: move it to feedback-link

File: sass/partials/_about-us.scss [hunk 7]
```diff
@@ -0,0 +1,265 @@
...
+        position: absolute;
+        left: 85px;
+    
+        content: "1/26";
+    
+        font-style: normal;
+    }
+    
+    &__feedback-link:hover {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_about-us.scss]: move it to feedback-link

File: sass/partials/_about-us.scss [hunk 8]
```diff
@@ -0,0 +1,265 @@
...
+            font-size: 77px;
+        }
+    
+        &__quote-text {
+            width: 350px;
+    
+            font-size: 14px;
+        }
+    
+        .feedback__text {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_about-us.scss]: are this class names correct?

File: sass/partials/_examples.scss [hunk 9]
```diff
@@ -0,0 +1,264 @@
...
+    }
+    
+    &__types {
+        color: #2d2829;
+    
+        font-family: "Circe Regular";
+        font-size: 12px;
+    }
+    
+    & {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_examples.scss]: remove redundant style block

File: sass/partials/_examples.scss [hunk 10]
```diff
@@ -0,0 +1,264 @@
...
+        }
+    
+        &__card {
+            width: 290px;
+            height: 185px;
+            margin: 0;
+            padding: 20px 0 0 20px;
+        }
+    
+        &__card--classic {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_examples.scss]: move all modifiers to card element style block

File: sass/partials/_examples.scss [hunk 11]
```diff
@@ -0,0 +1,264 @@
...
+        &__name {
+            font-size: 14px;
+            font-weight: 700;
+        }
+    
+        &__types {
+            font-size: 12px;
+        }
+    
+        &__name::after {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_examples.scss]: move to name style block

File: sass/partials/_examples.scss [hunk 12]
```diff
@@ -0,0 +1,264 @@
...
+            width: 288px;
+            height: 184px;
+        }
+    
+        &__name--big::after {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_examples.scss]: move to name style block

File: sass/partials/_examples.scss [hunk 13]
```diff
@@ -0,0 +1,264 @@
...
+            height: 388px;
+    
+            content: "";
+    
+            background-color: rgba(0, 0, 0, 0.1);
+        }
+    }
+    
+    @media(min-width: 1080px) {
+        &{
```
Comment (uladzislau-kirychenka) [inline sass/partials/_examples.scss]: redundant style block

File: sass/partials/_examples.scss [hunk 14]
```diff
@@ -0,0 +1,264 @@
...
+            padding-top: 0;
+        }
+        &__title {
+            font-size: 40px;
+        }
+    
+        &__card--classic {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_examples.scss]: move to card style block

File: sass/partials/_examples.scss [hunk 15]
```diff
@@ -0,0 +1,264 @@
...
+    
+        &__card {
+            position: relative;
+            width: 485px;
+            height: 315px;
+            margin: 0;
+            padding: 20px 0 0 20px;
+        }
+    
+        &__card--big {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_examples.scss]: move to card style block

File: sass/partials/_examples.scss [hunk 16]
```diff
@@ -0,0 +1,264 @@
...
+            width: 695px;
+            height: 655px;
+        }
+    
+        &__card-container {
+            height: 655px;
+        }
+    
+        &__name::after {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_examples.scss:238]: move to name style block

File: sass/partials/_examples.scss [hunk 17]
```diff
@@ -0,0 +1,264 @@
...
+            width: 485px;
+            height: 315px;
+        }
+    
+        &__name--big::after {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_examples.scss]: move to name style block

File: sass/partials/_examples.scss [hunk 18]
```diff
@@ -0,0 +1,264 @@
...
+        }
+        &__name{
+            font-size: 24px;
+            font-weight: 400;
+        }
+        &__types{
+            font-size: 20px;
+            color: #6c6869;
+        }
+        &__types::after{
```
Comment (uladzislau-kirychenka) [inline sass/partials/_examples.scss]: move to types style block

File: sass/partials/_examples.scss [hunk 19]
```diff
@@ -0,0 +1,264 @@
...
+            position: absolute;
+            top: 280px;
+            right: 20px;
+            content: '';
+            width: 20px;
+            height: 17px;
+            background: url(../assets/img/arrow.png) no-repeat;
+        }
+        &__types--lg::after{
```
Comment (uladzislau-kirychenka) [inline sass/partials/_examples.scss]: move to types style block

File: sass/partials/_examples.scss [hunk 20]
```diff
@@ -0,0 +1,264 @@
...
+            width: 133px;
+            top: 600px;
+            content: 'Смотреть';
+            right: 60px;
+            background-position-x: 115px;
+            background-position-y: 5px;
+        }
+        &__types--child::after{
```
Comment (uladzislau-kirychenka) [inline sass/partials/_examples.scss]: move to types style block

File: sass/partials/_footer.scss [hunk 21]
```diff
@@ -0,0 +1,126 @@
+.footer {   
+    & {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_footer.scss]: redundant style block

**Review by uladzislau-kirychenka — CHANGES_REQUESTED** [review]

File: sass/partials/_about-us.scss (hunk 1 above)
Comment (ossset) [reply sass/partials/_about-us.scss]: done

File: sass/partials/_care.scss (hunk 2 above)
Comment (ossset) [reply sass/partials/_care.scss]: done

File: sass/partials/_examples.scss (hunk 3 above)
Comment (ossset) [reply sass/partials/_examples.scss]: done

File: sass/partials/_examples.scss (hunk 4 above)
Comment (ossset) [reply sass/partials/_examples.scss]: done

File: sass/partials/_care.scss (hunk 5 above)
Comment (ossset) [reply sass/partials/_care.scss]: done

File: sass/partials/_examples.scss (hunk 17 above)
Comment (ossset) [reply sass/partials/_examples.scss]: done

File: sass/partials/_examples.scss (hunk 18 above)
Comment (ossset) [reply sass/partials/_examples.scss]: done

File: sass/partials/_examples.scss (hunk 19 above)
Comment (ossset) [reply sass/partials/_examples.scss]: done

File: sass/partials/_examples.scss (hunk 20 above)
Comment (ossset) [reply sass/partials/_examples.scss]: done

File: sass/partials/_footer.scss (hunk 21 above)
Comment (ossset) [reply sass/partials/_footer.scss]: done

File: sass/partials/_about-us.scss [hunk 22]
```diff
@@ -0,0 +1,335 @@
+.about-us {
+    & {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_about-us.scss]: remove redundant style block

File: sass/partials/_about-us.scss [hunk 23]
```diff
@@ -0,0 +1,335 @@
...
+        @include flex(column, flex-start, space-around);
+        height: 140px;
+        margin-left: 35px;
+    }
+
+    .social-icon:active {
+        transform: scale(1.1);
+    }
+
+    .vk {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_about-us.scss]: make this class name more specific

File: sass/partials/_about-us.scss [hunk 24]
```diff
@@ -0,0 +1,335 @@
...
+
+        &__people {
+            display: none;
+
+            &-md {
+                display: block;
+                padding-top: 120px;
+            }
+
+            &-md-wrapper {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_about-us.scss]: nest into &-md

File: sass/partials/_about-us.scss [hunk 25]
```diff
@@ -0,0 +1,335 @@
...
+                text-align: left;
+            }
+        }
+
+        &__confectioner-title {
+            margin: 0;
+            font-size: 26px;
+        }
+
+        &__title--md {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_about-us.scss]: nest in &__title

File: sass/partials/_about-us.scss [hunk 26]
```diff
@@ -0,0 +1,335 @@
...
+                top: 40px;
+            }
+        }
+
+        &__confectioner {
+            &-title {
+                font-size: 35px;
+            }
+
+            &-text-title {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_about-us.scss]: nest in text

File: sass/partials/_care.scss [hunk 27]
```diff
@@ -0,0 +1,262 @@
...
+        &__subtitle {
+            font-size: 35px;
+        }
+
+        &__description {
+            font-size: 20px;
+
+        }
+
+        &__title::after {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_care.scss]: nest in &__title

File: sass/partials/_examples.scss [hunk 28]
```diff
@@ -0,0 +1,306 @@
...
+
+    &__types {
+        color: #2d2829;
+
+        font-family: $mainFont;
+        font-size: 12px;
+    }
+
+
+    padding: 60px 0 50px 0;
```
Comment (uladzislau-kirychenka) [inline sass/partials/_examples.scss:85]: if this style belongs to .examples classname move it at the top

File: sass/partials/_footer.scss [hunk 29]
```diff
@@ -0,0 +1,150 @@
...
+        &__social-lg {
+            display: block;
+        }
+
+        &__social-md {
+            @include hide();
+        }
+
+
+        padding-top: 0;
```
Comment (uladzislau-kirychenka) [inline sass/partials/_footer.scss:129]: move it to the top

File: sass/partials/_individual.scss [hunk 30]
```diff
@@ -0,0 +1,709 @@
...
+            width: 6px;
+            height: 6px;
+
+            transition: 0.1s ease-in-out;
+
+            border-radius: 20px;
+            background-color: #ddd;
+        }
+
+        &-link--last {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_individual.scss:22]: nest --last and hover

File: sass/partials/_individual.scss [hunk 31]
```diff
@@ -0,0 +1,709 @@
...
+            transform: scale(1.1);
+
+            background-color: #f50049;
+        }
+    }
+
+
+
+
+    padding-bottom: 20px;
```
Comment (uladzislau-kirychenka) [inline sass/partials/_individual.scss:34]: move it to the top

File: sass/partials/_individual.scss [hunk 32]
```diff
@@ -0,0 +1,709 @@
...
+        font-size: 22px;
+    }
+
+    &__card {
+        margin-bottom: 10px;
+        padding: 8px 20px 20px 20px;
+
+        background-color: #fff;
+
+        &-number-text {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_individual.scss:64]: duplicate selector

File: sass/partials/_individual.scss [hunk 33]
```diff
@@ -0,0 +1,709 @@
...
+
+        margin: 0 auto;
+        margin-top: 20px;
+    }
+
+    &__item--md {
+        @include hide();
+    }
+
+    .image-wrapper::after {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_individual.scss:109]: nest into .image-wrapper

File: sass/partials/_individual.scss [hunk 34]
```diff
@@ -0,0 +1,709 @@
...
+        &-circle {
+            margin-bottom: 45px;
+
+            text-align: center;
+
+            font-family: $mainFont;
+            font-size: 14px;
+        }
+
+        &-text--design {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_individual.scss:147]: nest into -text

File: sass/partials/_individual.scss [hunk 35]
```diff
@@ -0,0 +1,709 @@
...
+        font-weight: 700;
+    }
+
+    &__button {
+        margin-top: 20px;
+        @extend %button;
+        @extend %button--small
+    }
+
+    .circle-img {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_individual.scss:175]: nest all .circle classnames below

File: sass/partials/_individual.scss [hunk 36]
```diff
@@ -0,0 +1,709 @@
...
+
+        color: #000;
+
+        font-size: 14px;
+        font-weight: 700;
+    }
+
+
+
+    .desing-img {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_individual.scss:201]: is this typo? and nest after into this

File: sass/partials/_individual.scss [hunk 37]
```diff
@@ -0,0 +1,709 @@
...
+            &--container {
+                display: block;
+            }
+
+            &-number-text {
+                margin: 0;
+                padding: 0;
+            }
+
+            &-number-text--last {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_individual.scss:422]: nest into -number-text

File: sass/partials/_individual.scss [hunk 38]
```diff
@@ -0,0 +1,709 @@
...
+                background: url(../assets/img/illustartion-md.png) no-repeat;
+            }
+
+            &--last {
+                @include flex(row, flex-start, flex-end);
+                width: 100%;
+            }
+
+            &--last::before {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_individual.scss]: nest into &--last

File: sass/partials/_individual.scss [hunk 39]
```diff
@@ -0,0 +1,709 @@
...
+                width: 120px;
+                height: 140px;
+
+                content: "";
+
+                background: url(../assets/img/illustartion-md.png);
+            }
+        }
+
+        .image-wrapper::after {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_individual.scss]: nest into image-wrapper

File: sass/partials/_individual.scss [hunk 40]
```diff
@@ -0,0 +1,709 @@
...
+        }
+
+        &__slider-container {
+            @include flex();
+        }
+    }
+
+    @media (min-width: 1080px) {
+
+        &__card--last::before {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_individual.scss]: nest into &__card

File: sass/partials/_individual.scss [hunk 41]
```diff
@@ -0,0 +1,709 @@
...
+            font-size: 35px;
+        }
+
+
+        &__card {
+            height: 575px;
+            margin: 20px auto;
+            max-width: 1220px;
+
+            &-number-text {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_individual.scss:566]: nest into &-number

File: sass/partials/_individual.scss [hunk 42]
```diff
@@ -0,0 +1,709 @@
...
+
+            &-text {
+                border-radius: 0;
+                border: none;
+                border-bottom: 1px solid;
+                width: 250px;
+                text-align: left;
+            }
+
+            &-text--small {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_individual.scss:682]: nest into &-text

File: sass/partials/_promo.scss [hunk 43]
```diff
@@ -0,0 +1,201 @@
...
+    &__cons-text {
+        font-family: $mainFont;
+    }
+
+    &__cons-container {
+        max-width: 300px;
+        margin: 0 auto;
+    }
+
+    background: $mainBodyColor;
```
Comment (uladzislau-kirychenka) [inline sass/partials/_promo.scss:18]: move this style to the top

File: sass/partials/_promo.scss [hunk 44]
```diff
@@ -0,0 +1,201 @@
...
+        margin: 0;
+        padding-bottom: 20px;
+
+        text-align: center;
+
+        font-family: $curvyFont;
+        font-size: 22px;
+    }
+
+    &__cons-text {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_promo.scss]: join cons-text styles with cons-omg into &__cons block

File: sass/partials/_promo.scss [hunk 45]
```diff
@@ -0,0 +1,201 @@
...
+            padding-left: 50px;
+
+            text-align: left;
+
+            font-size: 60px;
+            line-height: 55px;
+        }
+
+
+        position: relative;
```
Comment (uladzislau-kirychenka) [inline sass/partials/_promo.scss:88]: move this style to the top

File: sass/partials/_promo.scss [hunk 46]
```diff
@@ -0,0 +1,201 @@
...
+
+        &__bg {
+            padding-top: 100px;
+        }
+
+        &__img {
+            @include hide();
+        }
+
+        &__cons-img {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_promo.scss]: nest into &__cons

File: sass/partials/_pros.scss [hunk 47]
```diff
@@ -0,0 +1,161 @@
...
+    }
+
+    &__subtitle {
+        text-align: center;
+
+        font-family: $curvyFont;
+        font-size: 22px;
+    }
+
+    &__item-heading {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_pros.scss]: nest into &__item below

File: sass/partials/_pros.scss [hunk 48]
```diff
@@ -0,0 +1,161 @@
...
+            background-color: $mainBodyColor;
+            @include flex(column, flex-end, center);
+            padding-right: 60px;
+        }
+
+
+        position: relative;
+
+
+        &__item--ingridients::after {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_pros.scss]: nest into &__items

File: sass/partials/_pros.scss [hunk 49]
```diff
@@ -0,0 +1,161 @@
...
+
+            &-text {
+                width: 400px;
+            }
+
+            &-wrapper {
+                padding-left: 30px;
+            }
+
+            &-wrapper--ingridients {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_pros.scss]: nest into &-wrapper

File: sass/partials/_pros.scss [hunk 50]
```diff
@@ -0,0 +1,161 @@
...
+                padding-left: 40px;
+            }
+
+            &--ingridients::after {
```
Comment (uladzislau-kirychenka) [inline sass/partials/_pros.scss:147]: nest into &--ingridients which we'll be created after fixing the comment below

**Review by uladzislau-kirychenka — CHANGES_REQUESTED** [review]

File: sass/partials/_about-us.scss (hunk 22 above)
Comment (ossset) [reply sass/partials/_about-us.scss]: done

File: sass/partials/_about-us.scss (hunk 23 above)
Comment (ossset) [reply sass/partials/_about-us.scss]: done with about-us__icon-vk

File: sass/partials/_about-us.scss (hunk 24 above)
Comment (ossset) [reply sass/partials/_about-us.scss]: done

File: sass/partials/_about-us.scss (hunk 25 above)
Comment (ossset) [reply sass/partials/_about-us.scss]: done

File: sass/partials/_about-us.scss (hunk 26 above)
Comment (ossset) [reply sass/partials/_about-us.scss]: done

File: sass/partials/_promo.scss (hunk 46 above)
Comment (ossset) [reply sass/partials/_promo.scss]: done

File: sass/partials/_pros.scss (hunk 48 above)
Comment (ossset) [reply sass/partials/_pros.scss]: done

File: sass/partials/_pros.scss (hunk 49 above)
Comment (ossset) [reply sass/partials/_pros.scss]: done

File: sass/partials/_pros.scss (hunk 50 above)
Comment (ossset) [reply sass/partials/_pros.scss:147]: done

**Pull request merged**